atom/common/native_mate_converters/net_converter.cc

```diff
@@ -239,8 +239,10 @@ void GetUploadData(base::ListValue* upload_data_list,
     if (reader->AsBytesReader()) {
       const net::UploadBytesElementReader* bytes_reader =
           reader->AsBytesReader();
-      std::unique_ptr<base::Value> bytes(base::Value::CreateWithCopiedBuffer(
-          bytes_reader->bytes(), bytes_reader->length()));
+      auto vec =
+          std::vector<char>(bytes_reader->bytes(),
+                            bytes_reader->bytes() + bytes_reader->length());
+      auto bytes = std::make_unique<base::Value>(vec);
```

Comment:
@codebytere Can you inline the call to `std::vector<char>` here so that `base::Value(BlobStorage&&)` is invoked instead of `base::Value(BlobStorage&)`